docs/configuration.md

```diff
@@ -37,7 +37,9 @@ Description for this content. It may span multiple lines and be up to 4000 chara
 
 #### files
 
-Project-relative paths of the files to be included in the deployment. Wildcards are accepted, using [`.gitignore` syntax](https://git-scm.com/docs/gitignore).
+Project-relative paths of the files to be included in the deployment.
+Uses [`.gitignore` syntax](https://git-scm.com/docs/gitignore) for file include
+patterns, like wildcards.
```

Comment:
```suggestion
Uses [`.gitignore` syntax](https://git-scm.com/docs/gitignore) for file include
patterns, like wildcards. For example, `/some_dir/util.py` references a `util.py` file under the directory `some_dir` that is just under the root of the project to be published.
```

We might need to expand / explain more what "root of the project to be published" means (or link to the relevant section in our docs). But giving a concrete example of the syntax most likely to be confusing would be great.

Reply:
Perhaps we should also consider including an example when the forward slash is not used, and the pattern will match any file located in the project subdirectories?

Reply:
I expanded this a bit to include an example project structure, and a wildcard example. Let me know what you think

Reply:
Maybe something like the below?

"- `app.py` will match any file, in any directory, named `app.py`"